package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@oada/client",
-  "version": "2.0.0",
+  "version": "2.0.1",
```

Comment:
I think your change is worth at least a minor version bump :slightly_smiling_face:. We could even increment the major version since the interface of the `Config` option has changed (i.e., `_ws` to `connection`).